.github/workflows/reviewers.yml

```diff
@@ -7,41 +7,51 @@ on:
       - src/ontology/uberon-edit.obo
 
 jobs:
-  assign-reviewer:
+  check:
     runs-on: ubuntu-latest
-
+    outputs:
+      intersection_of_found: ${{ steps.check_intersection_of.outputs.intersection_of_found }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0  # Fetch the entire history for all branches
-      
+          fetch-depth: 0
+
       - name: Fetch base branch
         run: |
-          git fetch origin ${{ github.base_ref }}  # Fetch base branch explicitly
-      
+          git fetch origin ${{ github.base_ref }}
+
       - name: Check if equivalent class axiom was edited
         id: check_intersection_of
         run: |
           git diff origin/${{ github.base_ref }}...HEAD -- src/ontology/uberon-edit.obo > diff.txt
           if grep -E '^(-|\+)intersection_of:' diff.txt; then
-            echo "intersection_of_found=true" >> $GITHUB_ENV
+            echo "intersection_of_found=true" >> $GITHUB_OUTPUT
           else
-            echo "intersection_of_found=false" >> $GITHUB_ENV
+            echo "intersection_of_found=false" >> $GITHUB_OUTPUT
           fi
 
-      - name: Assign reviewer
-        if: env.intersection_of_found == 'true'
+      - name: Block PR by requesting changes
+        if: steps.check_intersection_of.outputs.intersection_of_found == 'true'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          gh api repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers \
-            --method POST \
-            --field reviewers[]=cmungall
+          gh pr review ${{ github.event.pull_request.number }} --request-changes --body "Changes detected in \`src/ontology/uberon-edit.obo\` involving \`intersection_of\`. Review by specific Uberon Core Team member is required."
 
-      - name: Block PR by requesting changes
-        if: env.intersection_of_found == 'true'
+
+  assign-reviewer:
+    needs: check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        reviewer: [ "cmungall", "dosumis", "aleixpuigb", "Caroline-99" ]
```

Comment:
Not sure I agree with this set of reviewers - unless everyone has had special training in logical reasoning, this defeats the purpose of this specific check. I know @aleixpuigb and @Caroline-99 are both highly capable curators and engineers, but have they been trained to really understand the responsibility and dangers associated with changes affecting the logical definitions of high-level classes? If so, then consider my comment void - I am of course _not_ blocking this PR if they are aware and trained in this matter.

I would suggest to add @gouttegd @balhoff and myself to this list as we have at least some level of training here.

Reply:
> Not sure I agree with this set of reviewers

The list was proposed by @dosumis in the Uberon call of June 16th, and approved there.

Happy to add you (and @balhoff if he doesn’t object) to the list.